client/my-sites/plans/jetpack-plans/product-store/features-list/style.scss

```
@@ -3,6 +3,10 @@
 
 .features-list {
 	padding: 1.5rem;
+	/* stylelint-disable scales/radii */
+	border-bottom-left-radius: 8px;
+	border-bottom-right-radius: 8px;
```

Can we please use [RTL-aware](https://hacks.mozilla.org/2015/10/building-rtl-aware-web-apps-websites-part-2/) styles instead of `left` and `right`?

```suggestion
	border-end-start-radius: 8px;
	border-end-end-radius: 8px;
```